docs/documentation_styleguide.rst

```diff
@@ -0,0 +1,234 @@
+========================
+Documentation Styleguide
```

Comment:
Style Guide is two words, unless you are creating a brand, like Styleguide™.

Also consider using the file name "documentation_style_guide.rst", if you decide to use two words instead of one.